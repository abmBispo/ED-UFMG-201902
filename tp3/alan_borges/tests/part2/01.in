8
capitulo lxxx venhamos ao capitulo venhamos ao capitulo
c ao
c capitulo
c lxxx
c venhamos
