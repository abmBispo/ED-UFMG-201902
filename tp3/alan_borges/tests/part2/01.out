01
11
00
10
